db/sg_replicate_conflict_resolver.go

```diff
@@ -137,6 +140,38 @@ func (c *ConflictResolver) Resolve(ctx context.Context, conflict Conflict) (winn
 	return winner, ConflictResolutionMerge, err
 }
 
+// ResolveForHLV is a wrapper for ConflictResolverFunc that evaluates whether conflict resolution resulted in localWins,
+// remoteWins, or merge, specifically for HLV-aware conflict resolution.
+func (c *ConflictResolver) ResolveForHLV(ctx context.Context, conflict Conflict) (winner Body, resolutionType ConflictResolutionType, err error) {
+
+	winner, err = c.crf(ctx, conflict)
+	if err != nil {
+		return winner, "", err
+	}
+
+	winningRev, ok := winner[BodyCV]
+	if !ok {
+		c.stats.ConflictResultMergeCount.Add(1)
+		return winner, ConflictResolutionMerge, nil
+	}
```

Comment:
What are the cases where this gets hit vs the end of this function

Reply:
The aim here was to keep in sync with current existing conflict resolution functions.